src/system_info/system_info_display_wayland.cc

```diff
@@ -204,7 +204,7 @@ bool SysInfoDisplay::UpdateBrightness() {
   int val = atoi(str_val);
   free(str_val);
 
-  brightness_ = val / max_val;
```

Comment:
Division of two integers give us integer so variable brightness_ is set only to 1 or 0. I think that expected brightness_ value is between 0 ... 1.